src/spyglass/decoding/v1/waveform_features.py

```diff
@@ -163,13 +171,14 @@ def make(self, key):
     def _fetch_waveform(
         merge_key: dict, waveform_extraction_params: dict
     ) -> si.WaveformExtractor:
-        curation_key = (SpikeSortingOutput.CurationV1 & merge_key).fetch1()
-        recording = CurationV1.get_recording(curation_key)
+        # get the recording from the parent table
+        recording = SpikeSortingOutput().get_recording(merge_key)
         if recording.get_num_segments() > 1:
             recording = si.concatenate_recordings([recording])
-        sorting = CurationV1.get_sorting(curation_key)
+        # get the sorting from the parent table
+        sorting = SpikeSortingOutput().get_sorting(merge_key)
 
-        waveforms_dir = temp_dir + "/" + str(curation_key["sorting_id"])
+        waveforms_dir = temp_dir + "/" + str(merge_key["merge_id"])
```

Comment:
Possible slight confusion here from setting `waveforms_dir` to be something other than what `spyglass.settings` thinks it is. Limited scope, so not a huge issue. Maybe `waveform_temp_dir`?

Reply:
Is there a waveforms_dir in spyglass.settings?